backend/app.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from flask import Flask, Response, jsonify
+from flask_jwt_extended import JWTManager
+
+from .api.auth import auth
+from .api.documents import document_api
+from .api.invites import invite_api
+from .api.messages import message_api
+from .http_exception import HTTPException
+from .services.email_sender import mail
+
+
+def create_flask_app() -> Flask:
+    application = Flask(__name__)
+
+    with open("config.json", "r") as config:
```

Comment:
No, you should set up config from environmental variables. You can access them through `os` module